signer/local/local_test.go

```diff
@@ -885,6 +887,69 @@ func TestNameWhitelistSign(t *testing.T) {
 
 }
 
+func TestExtensionSign(t *testing.T) {
```

Comment:
Can you add a test that validates the behaviour where by default, no extra extensions are permitted?